pkgs/development/python-modules/ftfy/default.nix

```diff
@@ -30,7 +31,10 @@ buildPythonPackage rec {
 
   propagatedBuildInputs = [ wcwidth ];
 
-  nativeCheckInputs = [ pytestCheckHook ];
+  nativeCheckInputs = [
+    versionInstallHook
```

Comment:
This should go in `nativeInstallCheckInputs`, which requires `doInstallCheck = true`

Reply:
> This should go in `nativeInstallCheckInputs`, which requires `doInstallCheck = true`

Yea you are sort of correct, but thinking about it the correct thing to do would be to rename the hook to `versionCheckHook`, because it is being run as part of the `distPhase`, and hence it has nothing to do with the `installCheckPhase`.

Reply:
> > This should go in `nativeInstallCheckInputs`, which requires `doInstallCheck = true`
> 
> Yea you are sort of correct, but thinking about it the correct thing to do would be to rename the hook to `versionCheckHook`, because it is being run as part of the `distPhase`, and hence it has nothing to do with the `installCheckPhase`.

Hmm on a 2nd thought, using the `distPhase` for this hook is a bit wierd, and it is not enabled by default in `stdenv`, and enabling `doDist = true;` will make the build try to run `make dist` which is also inappropriate if the goal is to run this test. I switched to `preInstallCheckHooks` eventually.